### PR TITLE
Scripts: add autopep8.sh

### PR DESCRIPTION
**Pull request by changsh726**

### CODE REVIEW

File: scripts/clang_format.sh [hunk 1]
```diff
@@ -39,7 +39,7 @@ function check_clang_format() {
 }
 
 function clang_format_run() {
-  ${CLANG_FORMAT_CMD} -i -style=Google "$@"
```
Comment (changsh726) [inline scripts/clang_format.sh:42]: This overrides the settings in `.clang-format`

File: scripts/buildifier.sh [hunk 2]
```diff
@@ -35,22 +35,35 @@ fi
...
       -exec ${BUILDIFIER_CMD} -lint=fix {} +
-  set +x
+    set +x
 
-else
-  error "Bazel files or directories expected, got '${FIRST_TARGET}'"
-  exit 1
-fi
+  else
+    error "Bazel files or directories expected, got '${FIRST_TARGET}'"
```
Comment (storypku) [inline scripts/buildifier.sh:64]: `FIRST_TARGET` should be renamed to `target`.

File: scripts/clang_format.sh (hunk 1 above)
Comment (storypku) [reply scripts/clang_format.sh:42]: > This overrides the settings in `.clang-format`

Better note this in this file.

File: scripts/apollo_format.sh [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+source "${TOP_DIR}/scripts/apollo.bashrc"
+
+function print_usage() {
+    info "Usage: $0 [Options] <path/to/src/dir/or/files> "
+    info "Options:"
+    info "${TAB}-p|--python      Format Python code"
+    info "${TAB}-b|--bazel       Format Bazel code"
+    info "${TAB}-c|--cpp         Format cpp code"
+    info "${TAB}-a|--all         Format all (C++/Python/Bazel)"
+    info "${TAB}-h|--help        # Show this message"
```
Comment (xiaoxq) [inline scripts/apollo_format.sh:35]: There are two indent styles in this pull request, 2-spaces and 4-spaces. Please stick to one style. And we can unify the whole repo later.

Comment (changsh726) [reply scripts/apollo_format.sh]: Ok

File: scripts/apollo_format.sh [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+#   apollo_format.sh [Options] <path/to/src/dir/or/files>
+
+# Fail on error
+set -e
+
+TOP_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+source "${TOP_DIR}/scripts/apollo.bashrc"
+
+function print_usage() {
+  info "Usage: $0 [Options] <path/to/src/dir/or/files> "
```
Comment (storypku) [inline scripts/apollo_format.sh:29]: `options` instead of `Options`. Please fix in your next PR.

File: scripts/apollo_format.sh [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    ;;
+  -a|--all)
+    run_format_all "$@"
+    ;;
+  -h|--help)
+    print_usage
+    exit 1
+    ;;
+  *)
+    echo "Unknown option: -$opt"
```
Comment (storypku) [inline scripts/apollo_format.sh:82]: `option` rather than `opt`

File: scripts/autopep8.sh [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+    error "Oops, autopep8 missing..."
+    error "Please make sure autopep8 is installed and check your PATH" \
+        "settings. For Debian/Ubuntu, you can run the following command:"
+    error "  sudo pip install --upgrade --no-cache-dir autopep8"
+    exit 1
+  fi
+}
+
+function autopep8_run() {
+  ${AUTOPEP8_CMD} -i -a -a "$@"
```
Comment (storypku) [inline scripts/autopep8.sh:55]: Two `-a` ?

Comment (changsh726) [reply scripts/autopep8.sh:55]: it means the aggressive level, multiple -a result in more aggressive changes.

**Pull request merged**